setup.py

```diff
@@ -17,53 +16,18 @@
         # switching between development branches
         subp.check_call(["git", "submodule", "update"])
 
-models = [
-    "eposlhc",
-    "sib21",
-    "sib23",
-    "sib23d",
-    "sib23c01",
-    "qgs01",
-    "qgsII03",
-    "qgsII04",
-    "pythia6",
-    "sophia",
-    "dpmjet306",
-    "phojet112",
-    "phojet191",
-    "phojet193",
-    "dpmjetIII191",
-    "dpmjetIII193",
-]
-
-# urqmd34 doesn't build correctly on Windows
-if platform.system() != "Windows":
-    models.append("urqmd34")
-    models.append("pythia8")
-
-# for convenience, support building extra models via extra.cfg
-# extra.cfg is not tracked by git, so can be freely modified
-# extra.cfg example:
+# for convenience, support building extra models via EXTRA
+# EXTRA is not tracked by git, so can be freely modified
+# EXTRA example:
 # -----
 # sib23c00
 # sib23c02
 # sib23c03
 # dev_dpmjetIII193=/full/path/to/dir/dpmjetIII-19.3
```

Comment:
Is the extra.cfg functionality gone?

Reply:
No, the comment just got messed up during development. Will be fixed.

Reply:
done